spec/src/main/asciidoc/required-apis.asciidoc

```diff
@@ -62,50 +68,21 @@ Use of implementations beyond CDI 3.0 in MicroProfile is allowed, however, not r
  - http://cdi-spec.org/
 
 [[jakarta-annotations]]
-=== Jakarta Annotations 2.0
+=== Jakarta Annotations 2.0 (TODO 2.1?)
```

Comment:
We should remove this section as well as we should just simply depend on Jakarta EE 10 Core Profile.